slacklib.py

```diff
@@ -138,20 +134,20 @@ def command_usage():
 
 def command_help():
     """Return the help string for slash commands."""
+    app_settings = models.AppSettings.get()
     return (
         "I can help you manage your "
-        "<{}|weekly snippets>! :pencil:".format(_WEB_URL) +
+        "<{}|weekly snippets>! :pencil:".format(app_settings.hostname) +
         command_usage()
     )
 
 
-def _no_user_error(user_email):
-    return (
-        "You don't appear to have a snippets account yet!\n"
-        "To create one, go to {}\n"
-        "We looked for your Slack email address: {}"
-        .format(_WEB_URL, user_email)
-    )
+def _no_user_error(user_id):
+    app_settings = models.AppSettings.get()
+
+    return (f"You don't seem to be logged in!  Please configure your slack "
```

Comment:
TODO: nothing stops users from putting in someone else's slack user ID for themselves, which would result in confusion and possibly a weird vector for harassment.  It's not a HUGE problem but it should be addressed at some point.